setup.cfg

```diff
@@ -75,6 +75,7 @@ install_requires =
     protobuf==3.20.3
     pycryptodome==3.15.0
     pyrsistent==0.19.2
+    python-dateutil==2.8.2
```

Comment:
I don't think this is used anymore? Can remove

Reply:
true